sandbox-web/src/main/java/com/exadel/sandbox/team5/DiscountRestController.java

```diff
@@ -21,8 +22,16 @@ public Discount getDiscount(@PathVariable Long id) {
     }
 
     @GetMapping("/all")
-    public List<Discount> getAll() {
-        return service.getAll();
+    public Page<Discount> getAll() {
+        Pageable pageWithFiveElements = PageRequest.of(0, 5);
+        return service.getAll(pageWithFiveElements);
+    }
+
+
+    @GetMapping("/all/{idTag}")
```

Comment:
I think that it's better to use url parameter (see `@QueryParam`) or pass some filter object as bod of POST request. Also `all` word in rest is redundant. Usually how use REST API:
`GET api/dsicounts/`  or with url params` - get all (or filtered objects)
`POST api/dsicounts/`  (where body is filter object) - get all or filtered objects
`GET api/dsicounts/{id}` - get one object
`GET api/dscounts/{id}/reviews/` - get reviews (some nested objects of discount)
`GET api/dscounts/{id}/reviews/{id}` - get one review object (some nested object of discount)